interview-questions.md

```diff
@@ -16,33 +16,33 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is a version control software
 
-Researched answer:
+Researched answer: Git is a technology that enables you to keep every codebase in sync with the remote codebase while all of them are changed frequently. 
 
 
 2. What is the difference between an array's index and its value?
 
-Your answer:
+Your answer: An array's value is the JS data type. The index is the place it holds in the array.
 
-Researched answer:
+Researched answer:Each individual value in an array is called an element, and each element has an index — an integer that denotes the element's place in the array.
 
 3. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
 
-Your answer:
+Your answer:Both arrays and strings are variables. Arrays are a variable that holds multiple values. A string is a variable that holds only letters, unlike an array that can hold numbers as well.
 
-Researched answer:
+Researched answer:Usually, the array size is fixed, while strings can have a variable number of elements. Arrays can contain any data type (char short int even other arrays) while strings are usually ASCII characters terminated with a NULL (0) character.
```

Comment:
Great research, something that will help drive home your point is using examples. Examples will be your best friend when explaining what you know. Don't be afraid to use them!